java/ql/src/Metrics/Summaries/TopJdkApis.qll

```diff
@@ -287,16 +287,19 @@ predicate hasApiName(Callable c, string apiName) {
 }
 
 /** A top JDK API. */
-class TopJdkApi extends SummarizedCallableBase {
+class TopJdkApi extends Callable {
   TopJdkApi() {
+    this.isSourceDeclaration() and
     exists(string apiName |
-      hasApiName(this.asCallable(), apiName) and
+      hasApiName(this, apiName) and
       topJdkApiName(apiName)
     )
   }
 
   /** Holds if this API has a manual summary model. */
-  private predicate hasManualSummary() { this.(SummarizedCallable).hasManualModel() }
+  private predicate hasManualSummary() {
+    exists(SummarizedCallable sc | sc.asCallable() = this and sc.hasManualModel())
+  }
 
   /** Holds if this API has a manual neutral model. */
   private predicate hasManualNeutral() {
```

Comment:
The original intention of this code was specific to neutral summaries (although I didn't make that clear in the QLDoc 😅), so I think it would be best to keep it that way for now.
```suggestion
  /** Holds if this API has a manual neutral summary model. */
  private predicate hasManualNeutralSummary() {
```
Line 306 would need to be updated to: 
```
this.(FlowSummaryImpl::Public::NeutralSummaryCallable).hasManualModel()
```
and line 310 to: 
```
predicate hasManualMadModel() { this.hasManualSummary() or this.hasManualNeutralSummary() }
```